core/utils/poly-look/cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseURL": "http://localhost:3000"
```

Comment:
cypress will create its own server if you don't say anything, but it works perfectly on static, already generated, pages.